### PR TITLE
keybase ping --gregor

### PR DESCRIPTION
**Pull request by oconnor663**
This is what's going to run in the docker integration tests.

### CODE REVIEW

File: go/libkb/ping_gregor.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	transport := &pingGregorTransport{
+		Contextified: NewContextified(g),
+		host:         strings.TrimPrefix(g.Env.GetGregorURI(), "fmprpc://"),
+	}
+	rpcHandler := &pingGregorHandler{
+		Contextified: NewContextified(g),
+		pingErrors:   make(chan error),
+		pingSuccess:  make(chan struct{}),
+	}
+	connection := rpc.NewConnectionWithTransport(rpcHandler, transport, keybase1.ErrorUnwrapper{}, true, keybase1.WrapError, g.Log, nil)
```
Comment (oconnor663) [inline go/libkb/ping_gregor.go]: It's possible that `NewConnectionWithTransport` was the wrong interface to use here, and that I should've used a simpler thing that wouldn't have needed all those methods above. If we want to extend this to test authentication later, though, I think we'll need a full connection and not just a one-off RPC?

File: go/libkb/ping_gregor.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
+package libkb
```
Comment (maxtaco) [inline go/libkb/ping_gregor.go]: can this go in client to keep additional `libkb` clutter?

**Comment by maxtaco** [comment]
One comment otherwise go!

**Pull request merged**